crates/configuration/src/version5/metadata/database.rs

```diff
@@ -0,0 +1,321 @@
+//! Metadata information regarding the database and tracked information.
+
+// This code was copied from a different place that predated the introduction of clippy to the
+// project. Therefore we disregard certain clippy lints:
+#![allow(
+    clippy::enum_variant_names,
+    clippy::upper_case_acronyms,
+    clippy::wrong_self_convention
+)]
```

Comment:
Feels appropriate to fix these when making breaking changes.

Reply:
fixed.